gtsam/hybrid/hybrid.i

```diff
@@ -105,14 +104,29 @@ class HybridBayesTree {
                  gtsam::DefaultKeyFormatter) const;
 };
 
+#include <gtsam/hybrid/HybridBayesTree.h>
```

Comment:
Umm why are we including this here (on HybridBayesNet) rather than before HybridBayesTree?

Reply:
Either way it doesn't matter because the wrapper will bubble it to the top of the generated files, but still seems like a weird thing to do (especially for wrapper noobs).